package.json

```diff
@@ -70,7 +70,7 @@
         "eslint-plugin-unused-imports": "^3.0.0",
         "eslint-plugin-workspaces": "^0.9.0",
         "husky": "^7.0.4",
-        "mocha": "^11.1.0",
+        "mocha": "11.7.2",
```

Comment:
Very minor... why we don't use `"^11.7.2"` to allow updates ?

Reply:
OK done, I also fixed a minor error raised by the "npm check:versions" steps